roles/openshift_health_checker/openshift_checks/efk_stack_check.py

```diff
@@ -0,0 +1,257 @@
+# pylint: disable=missing-docstring
+'''
+  Module for performing checks on an elasticseatch, fluentd, and kibana stack
+'''
+
+import json
+import os
+import ssl
+import urllib2
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EFKStackCheck(OpenShiftCheck):
+    """Module that checks an EFK stack shipped with OCP"""
+
+    name = "efk_stack_check"
+    tags = ["elasticsearch", "efkstack"]
+
+    logging_namespace_var = "openshift_logging_namespace"
+
+    def __init__(self, module_executor):
+        super(EFKStackCheck, self).__init__(module_executor)
+
+        es_cert_prefix = '/etc/elasticsearch/secret/admin-'
+        es_curl_base = "curl -s --cert {}cert --key {}key --cacert {}ca -XGET "
+        self.es_curl = es_curl_base.format(es_cert_prefix, es_cert_prefix, es_cert_prefix)
+
+    def run(self, tmp, task_vars):
+        es_pods, fluentd_pods = self.get_pods(task_vars)
+
+        status = dict()
+        status['elasticsearch'] = self.check_elasticsearch(es_pods, task_vars)
+        status['fluentd'] = self.check_fluentd(fluentd_pods, task_vars)
+        status['kibana'] = self.check_kibana(task_vars)
+
```

Comment:
Status check for curator looks to be missing.  Also need to consider how to handle the 'ops' components as well